extensions/amp-sidebar/1.0/amp-sidebar-1.0.md

```diff
@@ -58,103 +58,232 @@ limitations under the License.
 - Touch zoom is disabled on the `amp-sidebar` and it's mask when the sidebar is open.
 
 Example:
+
 ```html
-<amp-sidebar id='sidebar1' layout='nodisplay'>
+<amp-sidebar id="sidebar1" layout="nodisplay" side="right">
```

Comment:
I noticed you added "side=right" to this example and others, is this now a requirement?